tests/points/admin/test_category_admin.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import csv
+from io import StringIO
+
+from django.urls import reverse
+from django.core.files.uploadedfile import SimpleUploadedFile
+
+
+@pytest.mark.django_db
+class TestCategoryAdmin:
+    def create_csv_file(self, data):
+        csvfile = StringIO()
+        csv.writer(csvfile).writerows(data)
+        return csvfile.getvalue()
+
+    def test_uploading_file_with_missing_headers(self, client, superuser):
```

Comment:
Happy with the test, I don't think checking one missing header is enough

Reply:
added more tests to confirm that the upload fails for each of the missing required headers(name, longitude, latitude) and it does not fail when a non-required column in missing